lib/DDG/Spice/LeakDB.pm

```diff
@@ -22,7 +22,7 @@ triggers startend => 'hashme', 'leakdb';
 spice to => 'http://api.leakdb.abusix.com/?j=$1';
 spice wrap_jsonp_callback => 1;
 
-handle remainder_lc => sub{
+handle remainder => sub{
```

Comment:
@moollaza I removed the regex (you were right, it's already the remainder, so doesn't make sense to try to clean it up)

However, I'm not sure about the white space thing... wouldn't by "my super password" a valid password?